zipline/data/minute_bars.py

```diff
@@ -1078,21 +1086,42 @@ def get_last_traded_dt(self, asset, dt):
 
     def _find_last_traded_position(self, asset, dt):
         volumes = self._open_minute_file('volume', asset)
-        start_date_minutes = asset.start_date.value / NANOS_IN_MINUTE
-        dt_minutes = dt.value / NANOS_IN_MINUTE
+        start_date_minute = asset.start_date.value / NANOS_IN_MINUTE
+        dt_minute = dt.value / NANOS_IN_MINUTE
+
+        if asset.sid in self._known_zero_volume_dict:
```

Comment:
Using `try`/`KeyError` is preferable. Avoids an extra hash and lookup into dictionary.

e.g.

```
try:
    earliest_dt_to_search = self._known_zero_volume_dict[asset.sid] / NANOS_IN_MINUTE
except KeyError:
    earliest_dt_to_search = start_date_minute
```